src/askvm/lib/run.ts

```diff
@@ -36,24 +43,32 @@ export async function run(
   const { name } = code;
 
   if (name in values) {
-    const value = typed(values[name]);
-
-    if (value.type.name === 'code' && args) {
-      const code = value.value as AskCode;
-      return await run(options, code, args);
+    const value = values[name];
+    if (isAskCode(value)) {
+      // calling the resource referenced by the value with the arguments
+      return run(options, value as AskCode, args);
     }
 
     return value;
   }
 
   if (name in resources) {
     if (!(resources[name] instanceof Resource)) {
-      console.error('resource', resources[name]);
+      // console.error('resource', resources[name]);
```

Comment:
Please remove this line.

Reply:
I use it as a tool for debugging. Let's keep it unless we introduce proper debugging